test/mockData.js

```diff
@@ -12,7 +12,16 @@ const mockData = {
   user3: {
     username: 'naruto',
     email: 'naruto@gmail.com'
-  }
+  },
+  userA: {
+    username: 'fattylee',
+    email: 'fattty@gmail.com',
+  },
+  userB: {
+    username: 'unknown',
+    email: 'unknown@gmail.com',
+  },
+
```

Comment:
Why are we having user3, userA and userB. Can they be consistent

Reply:
I've implemented all the feedback. Thanks for taking time out to review my code.